src/data/faq_de.json

```diff
@@ -493,16 +493,29 @@
                         ]
                     },
                     {
-                        "title": "[Google/Android] Auf meinem Hauptbildschirm werde ich gewarnt, dass mein Gerät nicht oder nicht vollständig kompatibel ist. Was bedeutet das?",
-                        "anchor": "incompatibility_warning",
+                        "title": "[Google/Android] Eine Karte mit einer „Inkompatibilitätswarnung” auf meinem Hauptbildschirm benachrichtigt mich, dass mein Gerät nur teilweise kompatibel ist. Was bedeutet das?",

```

Comment:
Maybe use the term "Startseite" displayed under the house icon by the app instead of "Hauptbildschirm"?

Reply:
Both terms are used throughout the FAQ, but I agree that Startseite, as a term, is a bit better.